base/install_motor.sh

```diff
@@ -34,3 +36,30 @@ cd ..
 
 make -j${JOBS}
 make clean
+
+cd $EPICS_MODULES_PATH
+
+download_github_module dls-controls pmac $PMAC_VERSION
+
+rm pmac/configure/RELEASE.local.linux-x86_64
+rm pmac/configure/RELEASE.linux-x86_64.Common
+rm pmac/configure/CONFIG_SITE.linux-x86_64.Common
+
+echo "
+BUILD_IOCS=NO
+USE_GRAPHICSMAGICK=NO
+
+SSH             = $(pkg-config --variable=prefix libssh2)
+SSH_LIB         = $(pkg-config --variable=libdir libssh2)
+SSH_INCLUDE     = -I$(pkg-config --variable=includedir libssh2)
```

Comment:
In this case I'd either leave it as `pkg-config --cflags-only-I libssh2` (the empty output might change on some other distro? who knows) or empty.

Reply:
Agreed. Having it set by `pkg-config` seems better for me.